### ISSUE TITLE
Postal w/o MVC NuGet package? 

### ISSUE THREAD

**Issue by daedalus-codes** [andrewdavey/postal#167]
Would it be a possibility to separate the NuGet packages? eg Postal.Core and then Just the ViewResults & MVC specific related items separately?  

For instance, If I would like to just use Postal in a desktop app, but don't want to have the Microsoft.AspNet.Mvc package installed when just the RazorEngine dependency would suffice.

**Comment by bradanderson912** [comment]
Good idea! Splitting Postal into separate NuGet packages like Postal.Core and Postal.Mvc would give more flexibility for different project types such as [Western](https://thewesterngazette.co.uk/). This way, desktop apps could avoid unnecessary dependencies, keeping the setup lightweight with just RazorEngine.

**Comment by marklington817** [comment]
Awesome work thanks for sharing this [Repay balance](https://livinginemirates.com/rpay-balance-enquiry/) info with us i really like it.

**Comment by faithsmith5** [comment]
A staff outsourcing agency is essential for global businesses, offering access to diverse talent, simplifying labor law compliance, and overcoming cultural and language barriers. This is especially valuable for companies expanding into new markets or managing cross-border teams. Just like a[ cleaning company in Ajman ](https://wellexpertcleaning.com/cleaning-services-in-ajman/)ensures smooth operations in a home, outsourcing agencies help businesses run efficiently across borders while focusing on strategic goals.
